rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -695,19 +710,15 @@ public void eventLoggerLogPeginRejectionEvents_before_rskip_181_activation() thr
         PeginInstructionsProvider peginInstructionsProvider = mock(PeginInstructionsProvider.class);
         when(peginInstructionsProvider.buildPeginInstructions(any(BtcTransaction.class))).thenReturn(Optional.empty());
 
-        BridgeSupport bridgeSupport = new BridgeSupport(
-                bridgeConstants,
-                mockBridgeStorageProvider,
-                mockedEventLogger,
-                btcLockSenderProvider,
-                new PeginInstructionsProvider(),
-                mock(Repository.class),
-                executionBlock,
-                btcContext,
-                federationSupport,
-                btcBlockStoreFactory,
-                activations
-        );
+        BridgeSupport bridgeSupport = bridgeSupportBuilder
+                .bridgeConstants(bridgeConstants)
+                .provider(mockBridgeStorageProvider)
+                .eventLogger(mockedEventLogger)
```

Comment:
```suggestion
                .eventLogger(mockedEventLogger)
                .btcLockSenderProvider(btcLockSenderProvider)
```
Seems like you missed btcLockSenderProvided

Reply:
Thanks Marcos For Spotting That Out. Fixed And Resolved.